src/main/java/com/clouway/cuse/gae/GaeIndexRegistry.java

```diff
@@ -37,6 +38,22 @@ public void register(Object instance, IndexingStrategy strategy) {
   }
 
   @Override
+  public void registerAll(List<Object> instances, IndexingStrategy strategy) {
+    List<List<Object>> parts = Lists.partition(instances, 200);
```

Comment:
Comment explaining the logic with a link added and the 200 is now a MAX_BATCH_SIZE variable.